src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.cs

```diff
@@ -8,6 +8,9 @@ namespace System.IO.Compression
 {
     public static class ZipFile
     {
+        // Per the .ZIP File Format Specification 4.4.17.1 all slashes should be forward slashes
+        private const char c_pathSeparator = '/';
```

Comment:
huh, I could have sworn we used the c_ designator for private constants but indeed you are correct.  Would you like me to fix this up in TFS?